### ISSUE TITLE
Extend the README

### ISSUE THREAD

**Issue by deemp** [objectionary/normalizer#368]
@yegor256 suggested to extend the README ([link](https://github.com/objectionary/normalizer/issues/362#issuecomment-2127876055)) with:
- A short overview of the project
- A quick start

**Comment by yegor256** [comment]
@deemp please, don't forget to remove this information from the statically generated site, to avoid duplication

**Comment by deemp** [comment]
@yegor256, good point. I'm not sure we should avoid duplication here. Some people may prefer to read the site instead of the README. Hence, they may want to read the project overview and the quick start on the site.

I believe we can share some sections between README and the site. `mdbook` and `mdsh` (tools that we use) allow inlining files.

**Comment by yegor256** [comment]
@deemp sure, render it in both places is fine, as long as you don't duplicate the content inside the repo
